### PR TITLE
Does my civic duty

### LINKED ISSUE #19719: MiniStation Telecom
Hello, I have actually encountered a bug with Ministation, the telecom doesn't work anymore and the computers can't find the server for PDA and radio.

A french player with the SCP french section.

### PR DESCRIPTION
**Pull request by Bawhoppen**
Fixes https://github.com/tgstation/tgstation/issues/19719

I broke it so I gotta fix it

### CODE REVIEW

**Comment by Bawhoppen** [comment]
what the fuck why did travis fail

File: _maps/map_files/MiniStation/MiniStation.dmm [hunk 1]
```diff
@@ -14577,8 +14578,12 @@
 /turf/open/floor/plating/airless,
 /area/shuttle/escape)
 "Ht" = (
-/turf/closed/wall/mineral/titanium/overspace,
-/area/shuttle/escape)
+/obj/machinery/telecomms/broadcaster/preset_left/birdstation,
+/turf/open/floor/plasteel/black{
+	name = "Mainframe Floor";
+	initial_gas_mix = "n2=100;TEMP=80"
```
Comment (DrCelt) [inline _maps/map_files/MiniStation/MiniStation.dmm]: >variables in your map
that's just disgusting

Comment (Bawhoppen) [inline _maps/map_files/MiniStation/MiniStation.dmm]: its not my map and plus that was already like that fuck off

**Comment by Bawhoppen** [comment]
actually fuck

**Comment by Bawhoppen** [comment]
some weird business is going on

**Pull request closed without merging**